client/src/js/view/index.js

```diff
@@ -12,8 +12,10 @@ import { ModeChooser } from './components/modechooser';
 import { Clupi } from './components/clupi';
 import TouchHandler from "./components/TouchHandler";
 
-const view = (state, actions) => (
-    <TouchHandler state={state.msiAdmin} changeState={actions.setMsiAdmin}>
+const ADMIN_PINS = ["1337", "1940", "1985"];
+
+const MAIN_PAGE = (state, actions) => (
```

Comment:
why did you make the variables like constants when every other 'view' is a normal camelCase variable name